quickstart-mongodbatlas/ci/mongodb-master.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
This left over from the current quickstart or something required for taskcat?  I think the later but just want to check I got this.

Also we have an initiative to remove terms like master in this kind of use so if we can change it that would be awesome.